deploy/04_catalyst/0065_deploy_asset_upgrader.ts

```diff
@@ -13,8 +13,8 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const Asset = await deployments.get('Asset');
   const GemsCatalystsRegistry = await deployments.get('GemsCatalystsRegistry');
 
-  const {deployer, catalystAssetFeeRecipient} = await getNamedAccounts();
-
+  const { deployer, catalystAssetFeeRecipient } = await getNamedAccounts();
+  // TODO: verify audit comment
```

Comment:
What are fees values?

Reply:
I don't have them right now and I suspect they are not yet defined